specs/command-line-user-experience.md

```diff
@@ -288,6 +288,14 @@ lxc image import centos-7-x86\_64.tar.gz --created-at=2014-12-10 --expires-at=20
 lxc image import debian-jessie\_amd64.tar.gz dakara:                                                                    | Import a debian LXD image in the lxc image store of remote host "dakara"
 lxc image alias create centos/7 \<hash\>                                                                                | Create an alias for centos/7 pointing to our centos 7 image
 
+**Example output (lxc image list)**
+
+    HASH            PUBLIC    NAME                                  DESCRIPTION                     ALIAS
+    -----------------------------------------------------------------------------------------------------
+    146246146827... yes       busybox-amd64                         -                               busybox-amd64
+    95830b5e4e04... yes       ubuntu-vivid-amd64-default-20150308   Ubuntu 15.04 (devel) x86 64bit  ubuntu/devel (3 more)
```

Comment:
We can but I think it'd make things more confusing for users as you can start an image by its short hash but you can't by its name unless you pass name=ABC.

Also, name is a property so it's not guaranteed to be unique.